File: app/src/main/java/com/dev/orangebrowser/data/model/SiteCategory.java
package com.dev.orangebrowser.data.model;

import java.util.List;

public class SiteCategory {
    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    private String category;
    private List<Site> sites;

    public List<Site> getSites() {
        return sites;
    }

    public void setSites(List<Site> sites) {
        this.sites = sites;
    }


}

File: app/src/main/java/com/dev/orangebrowser/data/model/SiteCategory.kt
package com.dev.orangebrowser.data.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

/***
 *
 */

@Entity(tableName="site_category")
data class SiteCategory(
    @PrimaryKey(autoGenerate = true) var uid: Int,
    @ColumnInfo(name = "url") var url: String?, //首页
    @ColumnInfo(name = "name") var name: String?,//名称
    @ColumnInfo(name = "description") var description: String?,//描述
    @ColumnInfo(name = "icon") var icon: String?//icon
)